src/test/scala/mesosphere/marathon/api/v2/json/AppUpdateTest.scala

```diff
@@ -408,40 +380,6 @@ class AppUpdateTest extends UnitTest with ValidationTestLike {
       )), createdViaUpdate.container)
     }
 
-    "empty app persists existing residency" in {
```

Comment:
Hm, I think we still want to test whether the API accepts specs that do have the fields set.